brainscore_language/models/huggingface.py

```diff
@@ -34,18 +34,24 @@ def __init__(
         self.behavioral_task = None
         self.neural_recordings = []
 
+        self.task_function_mapping_dict = {
+            ArtificialSubject.Task.next_word: self.predict_next_word,
+        }
+
     def identifier(self):
         return self.model_id
 
     def perform_behavioral_task(self, task: ArtificialSubject.Task):
         self.behavioral_task = task
+        self.run_behavioral_experiment = self.task_function_mapping_dict[task]
```

Comment:
Since this function doesn't really "run" anything, but rather converts the output to something behaviorally meaningful, I propose renaming

```suggestion
        self.output_to_behavior = self.task_function_mapping_dict[task]
```

Reply:
done!